spring-integration-hazelcast/build.gradle

```diff
@@ -0,0 +1,26 @@
+apply plugin: 'java'
+apply plugin: 'maven'
+
+group = 'org.springframework.integration'
+
+description = 'Spring Integration Hazelcast Support'
+
+sourceCompatibility = 1.7
+targetCompatibility = 1.7
+
+
+
+repositories {
+        
+     maven { url "http://repo.maven.apache.org/maven2" }
+}
+
+dependencies {
+    compile group: 'org.springframework', name: 'spring-context', version:'4.1.5.RELEASE'
+    compile group: 'org.springframework.integration', name: 'spring-integration-core', version:'4.1.2.RELEASE'
+    compile group: 'com.hazelcast', name: 'hazelcast', version:'3.4.1'
+    compile group: 'com.hazelcast', name: 'hazelcast-spring', version:'3.4.1'
+    compile group: 'com.hazelcast', name: 'hazelcast-client', version:'3.4.1'
+    compile group: 'junit', name: 'junit', version:'4.12'
+    testCompile group: 'org.springframework', name: 'spring-test', version:'4.1.5.RELEASE'
```

Comment:
`spring-integration-test` would be better. It polls 'spring-test' transitively.
As well as the same for the `junit`.

Can you explain the reason of `hazelcast-client` dep?

Of course, we can come back to these concerns when you rework this `build.gradle` to the appropriate our style.
Or if you can't, let me know and I'll do it for you and you add my commit to your PR